vmdb/app/models/miq_report/search.rb

```diff
@@ -14,7 +14,7 @@ def get_limit_offset(page, per_page)
   end
 
   def get_sqltable(assoc)
-    r = self.db_class.reflect_on_association(assoc.to_sym)
+    r = db_class.reflection_with_virtual(assoc.to_sym)
```

Comment:
Well, before when using ActsAsArModel there aren't real reflections, because it wasn't a real class...everything was virtual.  What was this line *before* it got change to `reflect_on_association`?